x/auth/migrations/legacy/legacy.go

```diff
@@ -1,5 +1,6 @@
-package v040
+package legacy
```

Comment:
`legacy` basically means `<=0.42`, right? What do you think of keeping the name `v042` or `v040`? Coz v0.43 is also legacy

Reply:
yea happy to do this, I went with legacy to avoid confusion with removing 0.42 and below. Since they are mostly util items, should it migration/utils/legacy.go or should we go to v42?

Reply:
My preference would be v0.42, even though they are util items.

Then, usage would be `v042auth.AddrLen`, which seems clear and explicit.